airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/commands.py

```diff
@@ -3,14 +3,34 @@
 #
 
 import asyncclick as click
+import semver
 from pipelines.airbyte_ci.connectors.bump_version.pipeline import run_connector_version_bump_pipeline
 from pipelines.airbyte_ci.connectors.context import ConnectorContext
 from pipelines.airbyte_ci.connectors.pipeline import run_connectors_pipelines
 from pipelines.cli.dagger_pipeline_command import DaggerPipelineCommand
 
 
+class BumpType(click.ParamType):
+    name = "bump-type"
+
+    def __init__(self):
+        self.choices = ["patch", "minor", "major"]
```

Comment:
nit: wondering if there could be an enum with these values we could pulled from the `semver` lib